Makefile

```diff
@@ -525,3 +526,19 @@ test-e2e-cleanup: login-required
 	$(OC_CLI) delete restore -n $(OADP_TEST_NAMESPACE) --all --wait=false
 	for restore_name in $(shell $(OC_CLI) get restore -n $(OADP_TEST_NAMESPACE) -o name);do $(OC_CLI) patch "$$restore_name" -n $(OADP_TEST_NAMESPACE) -p '{"metadata":{"finalizers":null}}' --type=merge;done
 	rm -rf $(SETTINGS_TMP)
+
+EC ?= $(shell pwd)/bin/ec-$(shell go env GOOS)-$(shell go env GOARCH)
+EC_VERSION ?= 2.7.2
+
+.PHONY: editorconfig
+editorconfig:
+	@[ -f $(EC) ] || { \
+	set -e ;\
+	curl -sSLo $(EC).tar.gz https://github.com/editorconfig-checker/editorconfig-checker/releases/download/$(EC_VERSION)/ec-$(shell go env GOOS)-$(shell go env GOARCH).tar.gz ;\
```

Comment:
I would make this work offline. Something like https://github.com/openshift/oadp-operator/pull/1215/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R313

Reply:
The command linked does a path check that would only invoke curl if version is mismatched.
```
	if [ ! -f $(OPERATOR_SDK) ]; then \
```

You could totally run this offline (ie. not invoking curl) after you run it the first time.

Reply:
alright, understood. I think it already does that with first if, but will check again

Reply:
ok you're right.. it just maybe an incorrect version installed.

also instead of version checking like I did, maybe append version number to bin would be simpler.. then you'd only path check the `bin/EC-vXYZ` and good to go.